code/game/objects/items/tools/powertools.dm

```diff
@@ -35,6 +35,10 @@
 	tool_behaviour = TOOL_SCREWDRIVER
 	usesound = 'sound/items/drill_use.ogg'
 
+/obj/item/powertool/jaws_of_life/Initialize(mapload)
+	. = ..()
+	AddElement(/datum/element/eyestab)
```

Comment:
This is the wrong typepath, also it needs to consider that it only should eyestab in screwdriver mode.